news/2021-05-27-skinning-contest-results.md

```diff
@@ -0,0 +1,61 @@
+
+Offering some needed contrast to the dark-grey ambiance, light mountain and accent colours differentiate the skin from its minimalist counterparts. Additionally, the skin utilizes a rare gimmick - a fully-animated bottom section - that most other skins handle poorly. Yet, Krizto's skillful hands have taken the hearts of many skinning enthusiasts with this very trick. If you're tired of playing with static skins, Minimalist 2.0 will most certainly satisfy your animation-thirsty needs!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/InlwaD3GsJI" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Winner - FREEDOM DiVE↓
+
+Forged with love by the well-known osu! player [**BTMC**](https://osu.ppy.sh/users/3171691) and his team, [FREEDOM DiVE↓](https://osu.ppy.sh/community/forums/topics/1293207) takes the lead with **3705** votes, outscoring the 2nd place by just over twice as much! 
+
+Rather than simplicity and minimalism harboured by most other submissions, FREEDOM DiVE↓ features a vivid theme with colourful visual effects, carried out masterfully with aesthetic hand-drawn elements. Inspired by the song belonging to one of the most iconic beatmaps of all time, [xi - FREEDOM DiVE](https://osu.ppy.sh/beatmapsets/39804), the skin revolves around the little animate orb greeting the user on every screen with its cute expressions.
+
+Despite the extravagantly placed objects and richness in colour, BTMC team did a great job positioning, ensuring that none of the objects distract the player from clicking the circles. Appeasing the flamboyance-adorers with its well-designed artworks and animations, the skin guarantees you a pleasant and distraction-free gameplay session as well. Without a doubt, FREEDOM DiVE↓ is the perfect "osu! Champion approved" skin for those who like tagging along with the skin of their favourite osu! player.
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/nNIeBiWKYQI" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+With his FREEDOM DiVE↓ entry, BTMC is awarded the trophy worthy of a champion, a profile badge exclusively designed to be themed around the skin itself! Congratulations BTMC, well-deserved!
+
+![](/wiki/shared/news/2021-05-27-skinning-contest-results/skinning_contest_badge.jpg)
+
+---
+
+That said, it's time to pull up the anchor.
+
+Be sure to check out the [Skinning Contest results](https://osu.ppy.sh/community/contests/126) for a full listing of all 31 entries to see how well your favourites did or get a better picture of the voting.
```

Comment:
I think this is also fine as is? I don't even know any more.

Reply:
The "to" is sort of necessary to be more correct, but the comma is for pause/separation and not strictly necessary.

Reply:
Technically, both forms are grammatical to my knowledge. Adding "to" here will enable us to avoid the ambiguous reading of "how well your favourites get a better picture". However, such a reading wouldn't even sound sensible, meaning that there is no need to add another infinitive marker for semantic reasons. Comma usage, on the other hand, is totally optional as far as I see.

Reply:
If it becomes less ambiguous and more clear, and thus easier to read, I see no reason not to change it.